cmd/testutils/extractor/main.go

```diff
@@ -0,0 +1,38 @@
+package main
```

Comment:
It looks like `factoryState` is a TOML file. Instead of regexing over it, maybe we could just parse it out which would be easier to follow logically.

Reply:
@diligentis as part of #477, we are getting rid of the factoryState file. In order to keep the bash script working and have a way to get the factoryAddress after deployment we will probably be dumping to a temp file the output of the `deployment command`. So I think regex here will facilitate the extraction.

Reply:
Gotcha, makes sense @vtleonardo!